apps/src/util/HttpClient.ts

```diff
@@ -66,7 +66,30 @@ async function post(
   return response;
 }
 
+async function put(
```

Comment:
Thank you!

Reply:
I don't want to drag this out too much because it's already creeped into this PR but should there be tests for this function? (could also be added in a follow up PR)